### DISCUSSION TITLE
How to add custom TRIGGER notification

### DISCUSSION THREAD

**Discussion by plzm** [InteractionDesignFoundation/add-event-to-calendar-docs#44] [General]
How to add a custom notification, for example 60 minutes before the event, to a Google calendar hyperlink?

Reference: https://icalendar.org/iCalendar-RFC-5545/3-2-14-alarm-trigger-relationship.html

**Comment by rbogdanowicz** [comment]
Up!

**Comment by alies-dev** [comment]
Short answer for Google: there is no reliable way to do it through a hyperlink today.

There is one related parameter, `erem`, and it is repeatable. It does two things: it flips the event onto custom reminders instead of the calendar default, and it adds one notification row per occurrence. The format is three numbers separated by colons, and the third must not be `-1`, otherwise the entry is dropped.

The problem is the values. Re-tested on 2026-08-19 with `erem=45:1:0`, `erem=7:8:9`, `erem=1:60:0` and several other combinations: a notification row appears every time, but it always lands on 0 minutes with the method left at the default. The offset never reaches the UI. So `erem` is enough to switch the event to custom reminders, and not enough to say "60 minutes before".

For comparison, on Yahoo this does work. `rem1` and `rem2` take an offset before the event, and the value has to land exactly on one of the offsets the composer supports (`5M`, `15M`, `30M`, `1H`, `2H`, `3H`, `6H`, `12H`, `1D` through `14D`), otherwise it silently becomes "no reminder":

```
https://calendar.yahoo.com/?v=60&TITLE=Test&ST=20261231T193000&ET=20261231T223000&rem1=1H
```

Outlook's compose deep link has no reminder parameter at all.

If you need an exact `TRIGGER` on a Google event, the options are an ICS file with a `VALARM`, or the Calendar API `reminders.overrides`. Both are documented in `services/google.md` terms as out of scope for URL parameters.

`erem` is documented with this caveat in `services/google.md`.
